examples/python-venv/devenv.nix

```diff
@@ -0,0 +1,6 @@
+{ pkgs, ... }:
+
+{
+  languages.python.enable = true;
+  languages.python.venv.enable = true;
```

Comment:
I really like a bit more descriptive names like `createVirtualenv` since the expectation of what will happen is clearer.

Reply:
Should it have two options then? Once for automatically creating a virtualenv and one for activating?